### PR TITLE
BVAL-616 Making @Positive/@Negative#strict() by default

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: src/main/java/javax/validation/constraints/Negative.java [hunk 1]
```diff
@@ -58,7 +58,7 @@
 	 *
 	 * @return whether the annotated element must be strictly negative
 	 */
-	boolean strict() default false;
+	boolean strict() default true;
```
Comment (gsmet) [inline src/main/java/javax/validation/constraints/Negative.java]: You decided against renaming `strict()` to `orZero()`?

**Comment by gunnarmorling** [comment]
Yes, I'm not a big fan of orZero() tbh. But both works, so if there is a
majority for renaming it, we can do it.

**Comment by chkal** [comment]
I also prefer `strict` over `orZero`.

**Comment by gunnarmorling** [comment]
Waiting for the results of our poll before moving on.

**Comment by gunnarmorling** [comment]
Closing this, going to add another PR for the separate pair of annotations as per our poll and discussion.

**Pull request closed without merging**